packages/flutter/test/cupertino/route_test.dart

```diff
@@ -143,48 +143,80 @@ void main() {
 
   testWidgets('Leading auto-populates with back button with previous title', (WidgetTester tester) async {
     await tester.pumpWidget(
-      const CupertinoApp(
-        home: Placeholder(),
+      CupertinoApp(
```

Comment:
Instead of replacing an existing test, can you write a new testWidgets with this test, instead?